<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvestigationBar</class>
 <widget class="QFrame" name="InvestigationBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QPushButton" name="newPatientButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>New Patient</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="existingPatientButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Existing Patient</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="recordButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zeroPressureButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Zero Pressure</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sensationsButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>First Desire</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
